cli/server.go

```diff
@@ -111,6 +111,8 @@ func Server(newAPI func(*coderd.Options) *coderd.API) *cobra.Command {
 		autoImportTemplates              []string
 		spooky                           bool
 		verbose                          bool
+		// maxWorkspacesPerUser is a uint8 to ensure a number > 0.
+		maxWorkspacesPerUser uint8
```

Comment:
A hard maximum of 255 seems quite low, why not a uint or a uint64 so admins can do whatever they want?